CHANGELOG.md

```diff
@@ -1,5 +1,529 @@
+  - encode the max early data size into the session ticket
+  - implement parsing of the early_data extension in the EncryptedExtensions
+  - add a tls.Config.MaxEarlyData option to enable 0-RTT
+  - accept TLS 1.3 cipher suites in Config.CipherSuites
+  - introduce a function on the connection to generate a session ticket
+  - add a config option to enforce selection of an application protocol
+  - export Conn.HandlePostHandshakeMessage
+  - export Alert
+  - reject Configs that set MaxVersion < 1.3 when using a record layer
+  - enforce TLS 1.3 when using an alternative record layer
+- github.com/multiformats/go-multiaddr (v0.2.2 -> v0.3.1):
+  - dep: add "codependencies" for handling version conflicts ([multiformats/go-multiaddr#132](https://github.com/multiformats/go-multiaddr/pull/132))
+  - Support /p2p addresses encoded as CIDs ([multiformats/go-multiaddr#130](https://github.com/multiformats/go-multiaddr/pull/130))
+  - Merge go-multiaddr-net
+- github.com/multiformats/go-multiaddr-net (v0.1.5 -> v0.2.0):
+  - Deprecate ([multiformats/go-multiaddr-net#72](https://github.com/multiformats/go-multiaddr-net/pull/72))
+- github.com/multiformats/go-multihash (v0.0.13 -> v0.0.14):
+  - fix: only register one blake2s length ([multiformats/go-multihash#129](https://github.com/multiformats/go-multihash/pull/129))
+  - feat: add two filecoin hashes, without Sum() implementations ([multiformats/go-multihash#128](https://github.com/multiformats/go-multihash/pull/128))
+  - feat: reduce blake2b allocations by special-casing the 256/512 variants ([multiformats/go-multihash#126](https://github.com/multiformats/go-multihash/pull/126))
+- github.com/multiformats/go-multistream (v0.1.1 -> v0.1.2):
+  - upgrade deps + interoperable varints. (#51) ([multiformats/go-multistream#51](https://github.com/multiformats/go-multistream/pull/51))
+- github.com/multiformats/go-varint (v0.0.5 -> v0.0.6):
+  - fix minor interoperability issues. (#6) ([multiformats/go-varint#6](https://github.com/multiformats/go-varint/pull/6))
+- github.com/warpfork/go-wish (v0.0.0-20190328234359-8b3e70f8e830 -> v0.0.0-20200122115046-b9ea61034e4a):
+  - Add ShouldBeSameTypeAs checker.
+  - Integration test update for go versions.
+- github.com/whyrusleeping/cbor-gen (v0.0.0-20200123233031-1cdf64d27158 -> v0.0.0-20200402171437-3d27c146c105):
+  - Handle Nil values for cbg.Deferred ([whyrusleeping/cbor-gen#14](https://github.com/whyrusleeping/cbor-gen/pull/14))
+  - add name of struct field to error messages
+  - Support uint64 pointers ([whyrusleeping/cbor-gen#13](https://github.com/whyrusleeping/cbor-gen/pull/13))
+  - int64 support in map encoders ([whyrusleeping/cbor-gen#12](https://github.com/whyrusleeping/cbor-gen/pull/12))
+  - Fix uint64 typed array gen ([whyrusleeping/cbor-gen#10](https://github.com/whyrusleeping/cbor-gen/pull/10))
+  - Fix cbg self referencing import path ([whyrusleeping/cbor-gen#8](https://github.com/whyrusleeping/cbor-gen/pull/8))
+
+### Contributors
```

Comment:
Note for posterity: There were a couple modifications made to the mkreleaselog script for this run through:
1) Some code generated files from go-libp2p-asn-util were manually ignored as they didn't follow a nice pattern for detecting codegen'd files.
2) Commits to go-multiaddr were ignored do to the overwhelming number of commits that were ported from `go-multiaddr-net`. Nobody's numbers should have been affected in a meaningful way.